### PR TITLE
Make every obj/.../New() call it's parents, Fixes things not burning on lava and causing runtimes

### PR DESCRIPTION
**Pull request by farie82**
## What Does This PR Do
Makes all objects call their parents New
Fixes:
image

I would like a test merge for this just to be sure that nothing broke. 

"\/obj\/item(\/[A-Z_0-9]*)*\/New\(\)" was used to find all the News

## Why It's Good For The Game
Less runtimes when things burn on lava and just more expected behaviour and better code

## Changelog
:cl:
fix: All things now burn properly on lava
/:cl:

### CODE REVIEW

**Comment by Citinited** [comment]
Would it be better to convert all these `New()`s to `Initialize()`? Given we probably want to do that for everything eventually 😄

**Comment by farie82** [comment]
> Would it be better to convert all these `New()`s to `Initialize()`? Given we probably want to do that for everything eventually 😄

I did think about that. But that would also include changing the code below these and such... bit much for myself currently to do. But feel free to do it :D

**Comment by Citinited** [comment]
Ahh right, thanks for checking! If I were to do that it would be in a mass refactor :P

File: code/game/objects/items/weapons/melee/energy.dm [hunk 1]
```diff
@@ -220,7 +221,7 @@
 	light_color = LIGHT_COLOR_RED
 
 /obj/item/melee/energy/sword/pirate/New()
-	return
+	..()
```
Comment (Fox-McCloud) [inline code/game/objects/items/weapons/melee/energy.dm]: this shouldn't exist at all.

File: code/game/objects/items/weapons/storage/backpack.dm [hunk 2]
```diff
@@ -63,7 +63,6 @@
 
 /obj/item/storage/backpack/holding/New()
 	..()
-	return
```
Comment (Fox-McCloud) [inline code/game/objects/items/weapons/storage/backpack.dm]: same

File: code/game/objects/items/weapons/melee/energy.dm (hunk 1 above)
Comment (farie82) [reply code/game/objects/items/weapons/melee/energy.dm]: Darn forgot about those. Removed in the newest commit

**Pull request merged**